SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialEventTracer.h

```diff
@@ -11,11 +11,35 @@
 #include <WorkerSDK/improbable/c_io.h>
 #include <WorkerSDK/improbable/c_trace.h>
 
+#include "SpatialEventTracer.generated.h"
+
 // Documentation for event tracing in the GDK can be found here: https://brevi.link/gdk-event-tracing-documentation
 DECLARE_LOG_CATEGORY_EXTERN(LogSpatialEventTracer, Log, All);
 
+UENUM()
+namespace EEventTraceFileOutputType
```

Comment:
is this preferable to using an enum class?

Reply:
enum class is preferable but unfortunately isn't compatible with TEnumAsByte in configs - suggests `enum class E : uint8` but this still has compile errors

Reply:
Fair enough if you TEnumAsByte then there is no way around it.